### PR TITLE
Fix typo in carousel example

### PR DESCRIPTION
**Pull request by Zaki0xFF**

### CODE REVIEW

File: examples/carousel/ui/carousel.slint [hunk 1]
```diff
@@ -13,7 +13,7 @@ export component Carousel inherits FocusScope {
     in property <length> itemWidth;
     in property <int> count: 0;
     private property <length> center-x: (root.width - Theme.size-big) / 2;
-    private property <duration> duation: Theme.duration-regular;
+    private property <duration> duration: Theme.duration-regular;
```
Comment (tronical) [inline examples/carousel/ui/carousel.slint:16]: Well spotted! Thanks :) Could you please also fix the same typo in line 66, so that the file compiles?

**Review by tronical — CHANGES_REQUESTED** [review]

File: examples/carousel/ui/carousel.slint (hunk 1 above)
Comment (Zaki0xFF) [reply examples/carousel/ui/carousel.slint:16]: No problem :)

**Review by tronical — APPROVED** [review]
Thanks for the quick fix :)

**Comment by Zaki0xFF** [comment]
My pleasure :)

**Pull request merged**